src/VisualStudio/Core/Def/Implementation/ProjectSystem/VisualStudioProject.cs

```diff
@@ -574,9 +574,10 @@ public void AddSourceFile(string fullPath, SourceCodeKind sourceCodeKind = Sourc
             string fullPath,
             SourceCodeKind sourceCodeKind = SourceCodeKind.Regular,
             ImmutableArray<string> folders = default,
+            bool designTimeOnly = default,
```

Comment:
This is potentially an API break, we've confirmed nobody uses it via IVT?

Reply:
I found no usages